src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -279,6 +279,12 @@
   <data name="Text_Loading" xml:space="preserve">
     <value>Loading...</value>
   </data>
+  <data name="Text_LoadingReadme" xml:space="preserve">
+    <value>Loading README...</value>
+  </data>
+  <data name="Text_ReadmeLoaded" xml:space="preserve">
+    <value>readme loaded</value>
```

Comment:
Why is the casing different here?

Reply:
Since Text_ReadmeLoaded isn't visible I figured I would set it all to lower case.

Reply:
As we discussed offline, it's not visible, but it does affect how screen readers read it. Thanks for updating it!